spec/integration/support/command_line_examples.rb

```diff
@@ -108,5 +108,20 @@
         end
       end
     end
+
+    describe "remove" do
+      context "when no arguments given" do
+        it "shows the help message for the remove command" do
```

Comment:
Do you want to test that the command fails or that the help is shown? Because right now the test does something different than the name says it does.